packages/web/public/config.yaml

```diff
@@ -39,10 +41,10 @@ timeLockAssets:
     contract: "0x9d633f05Ea918D392bDa0A74bD15E5bcCad324e2"
     standard: ERC20
     priceId: MTK
-    logo: https://staking.ktondao.xyz/images/token/kton.svg
+    logo: 
```

Comment:
@snoopy1412 Please give it a default token icon instead of the kton or usdt ones.